### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#568] [Galaxy Trivia] [answered]
🟨⬜🗝️
☯🟥🟦🟥
🦅🟨🟦🟥

**Comment by SumitKumarKar01** [comment]
VATICAN CITY
SOUTH KOREA
MOLDOVA

**Reply by ParanoidUser** [reply]
Nice try! For (2), follow the color lines. For (3), order is important.

**Reply by Veercodeprog** [reply]
Belarus
cambodia
ukraine

**Comment by ParanoidUser** [comment]
@aliabdallah7, @Taky0nDon, @alejandroch1202, @juanjpf we need your fresh ideas here. Can you guess? 🤓

**Comment by thuongtruong1009** [comment]
The first two symbols in the first row, "🟨⬜" could represent the colors of a flag, which are yellow and white. The key symbol "🗝️" might indicate some sort of important or historical location.

The second row contains the symbols "☯🟥🟦🟥", which could possibly represent the yin-yang symbol and the colors red, blue, and red.

The last row contains the symbols "🦅🟨🟦🟥", which might represent an eagle and the colors yellow, blue, and red.

Based on this interpretation, I would guess that the country being represented here is Mexico, as the flag of Mexico contains the colors yellow, white, red, and green, and the eagle is a national symbol. The key might represent the ancient Mayan or Aztec ruins found in the country.

**Comment by peterbaccaro** [comment, marked as answer]
1. VATICAN CITY
2. MONGOLIA
3. ECUADOR

![flags](https://user-images.githubusercontent.com/74666079/232550827-311be988-39aa-4eab-85c8-fa8d3cca8954.png)
